### ISSUE TITLE
include nanopore cDNA

### ISSUE THREAD

**Issue by michieitel** [Gaius-Augustus/BRAKER#53]
Hi!

How is it possible to include nanopore cDNA in the BRAKER2 pipeline in addition to RNAseq data if I have both?

Can I use both bam files or should the mapped nanopore data be in gff/gff3 format?

Thanks for your help
Michael

**Comment by KatharinaHoff** [comment]
Dear Michael,

if you aligned the nanopore data to genome, and if the resulting file is a
BAM file, you can provide that file to BRAKER; you can also convert the bam
file to hints with bam2hints and provide the output of this to BRAKER. (I
hope that will work; haven't tried with any nanopore-specific alignment
tool output file...)

BRAKER will likely have an issue with nanopore-only reads, because the
self-training GeneMark-ET approach has been optimized for high coverage
RNA-Seq data (e.g. Illumina). The information from a BAM file that is used
by BRAKER is only: splice-alignments, and which coverage do spliced
alignments have. It won't be a problem if you combine the nanopore data
with e.g. Illumina data.

Best,

Katharina

**Comment by michieitel** [comment]
Hi Katharina,

thanks for your answer. According to your recommenadations I ran BRAKER on the nanopore cDNA data (~17M reads) and the results are not that bad. Depending on the gene the gene models are very similar to the RNAseq based.

I also tried on the combined/merged bam file. However, this resulted in way worse gene models that either dataset on its own. Any suggesstions for the settings to improve the models on the combined data? I ran pretty much on defaults (--augustus_args="--protein=on" --gff3 on --cores 12 ).

Does it makes sense to run BRAKER on the nanopore data based on the parameters generated using RNAseq data rather than doing the training again?

Thanks for your help.
Michael

**Comment by michieitel** [comment]
Hi Katharina,
wondering if you could share your expertise regarding my question above.

Thanks
Michael

**Comment by KatharinaHoff** [comment]
Hi Michael,

yes, I would recommend using existing parameters for a species (e.g. from
training with RNA-Seq Illumina data) for new data in prediction (such as
Nanopore cDNA). Saves the time for training. Maybe have a look at the log
file from the original training, it lists some accuracy values (based on
benchmarking against a hold-out test set from the training data). If you
are happy with the values, there, keep the parameter set and only play with
prediction.

Best,
Katharina

**Comment by michieitel** [comment]
Great. Thanks Katharina!
Best,
Michael
